samples/snippets/publisher.py

```diff
@@ -146,31 +145,28 @@ def publish_messages_with_error_handler(project_id, topic_id):
 
     publisher = pubsub_v1.PublisherClient()
     topic_path = publisher.topic_path(project_id, topic_id)
+    publish_futures = []
 
-    futures = dict()
-
-    def get_callback(f, data):
-        def callback(f):
+    def get_callback(publish_future, data):
+        def callback(publish_future):
             try:
-                print(f.result())
-                futures.pop(data)
-            except:  # noqa
-                print("Please handle {} for {}.".format(f.exception(), data))
+                # Wait 100 ms for the publish call to succeed.
+                print(publish_future.result(timeout=0.1))
```

Comment:
Waiting only 100ms is likely to result in deadline exceeded failures. It's best not to set a deadline or if necessary, set a deadline closer to 60s.